alembic/versions/8e7282ae4840_updated_rtp_tables.py

```diff
@@ -0,0 +1,55 @@
+"""updated rtp tables
+
+Revision ID: 8e7282ae4840
+Revises: e66c069eb92b
+Create Date: 2017-08-13 18:06:40.892734+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '8e7282ae4840'
+down_revision = 'e66c069eb92b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('host_status')
```

Comment:
Actually I suppose not. That table's been superseded, but I didn't drop it from the Karoo so that we could preserve its data.

If Alembic is going to want to drop the table every time we change the DB schema, though, we should come up with a solution. Most sensible might just be to back up the database contents with `pg_dump` then drop the table from the running DB.